source/manual/alerts/RouterErrorRatioTooHigh.html.md

```diff
@@ -0,0 +1,35 @@
+---
+owner_slack: "#govuk-2ndline-tech"
+title: Router error ratio too high
+parent: "/manual.html"
+layout: manual_layout
+section: Pagerduty alerts
+---
+
+You can find the router request error rates on this dashboard:
+
+- [5xx Router Requests][router-5x https://grafana.eks.production.govuk.digital/d/uBEWkijnz/router-request-rates-errors-durations?orgId=1 x-request-rates-grafana]
+
+You can also view the 500+503 error rates across all applications on this dashboard:
+
+- [5xx app Requests][app-5xx-error-rates-grafana]
+
+## description
+
+This alert will fire when the ratio of requests in an error state are above the threshold configured in
+this config [file][https://github.com/alphagov/govuk-helm-charts/blob/main/charts/monitoring-config/rules/router.yaml]
+
+## impact
+
+Router deals with requests to the rails applications which means that both publishers and/or end users could be seeing errors depending on which application is returning errors.
+Use [this][app-5xx-error-rates-grafana] dashboard to check which applications have been erroring recently then look in [Kibana][prod-kibana] for more detailed look into the specific log messages.
+
+## potential resolution steps
+
+- If an application has been recently updated and is returning errors it may require a rollback using the app's deploy github action https://github.com/alphagov/<appname>/actions/workflows/deploy.yml
+- If your application is currently dealing with a sudden increase in load it could be that you need to scale your application. You can check the following [documentation][scale-app] in order to do this.
```

Comment:
Could simplify this to
```suggestion
- If your application is currently dealing with a sudden increase in load it could be that you need to [scale your application][scale-app].
```